linux_os/guide/services/ntp/service_timesyncd_configured/rule.yml

```
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Configure Systemd Timesyncd Servers'
+
+description: |-
+    <tt>systemd-timesyncd</tt> is a daemon that has been added for synchronizing the system clock
+    across the network. More information on <tt>chrony</tt> can be found at
+    The <tt>systemd-timesyncd</tt> daemon implements:
+      - Implements an SNTP client
+      - Runs with minimal privileges
+      - Saves the current clock to disk every time a new NTP sync has been acquired
+      - Is hooked up with networkd to only operate when network connectivity is available
+    Add or edit server or pool lines to <tt>{{{ chrony_conf_path }}}</tt> as appropriate:
+    <pre>server &lt;remote-server&gt;</pre>
+    Multiple servers may be configured.
+
+rationale: |-
+    If <tt>systemd-timesyncd</tt> configured ensures time synchronization is working properly.
```

This sentence is little messed up

```suggestion
    Configuring <tt>systemd-timesyncd</tt> ensures time synchronization is working properly.
```